src/main/controller/UserController.ts

```diff
@@ -224,6 +224,62 @@ export class UserController {
     }
   };
 
+  updateWorkout = async (req: Request, res: Response): Promise<any> => {
+    try {
+      // TODO: validate that the user with userId is logged in
+      // TODO: add tests for endpoint
+
+      req.body.userId = req.params.userId;
```

Comment:
Is there a reason why we're changing the `userId` associated with the given workout?

Reply:
`userId` actually isn't passed into the request body - it shows up as a param so I'm just setting it here so that I can easily do line 233!